bigbluebutton/greenlight
# Admin users delete

## Description

- Allows admin to delete specific users
- Fixes horizontal spacing issue on tables
- Adds icons to drop-down
## Testing Steps

- Pull changes
- Go to manage users in admin panel
- Click on three dots on any user row
- Click Delete
- Delete the user and make sure it works
## Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/38328371/178532566-535bf0d7-bdeb-4fe0-8b59-8a1c5156f02c.png)
